quinn-proto/src/frame.rs

```diff
@@ -120,42 +120,21 @@ pub enum Frame {
     ConnectionClose(ConnectionClose),
     ApplicationClose(ApplicationClose),
     MaxData(u64),
-    MaxStreamData {
-        id: StreamId,
-        offset: u64,
-    },
+    MaxStreamData { id: StreamId, offset: u64 },
     MaxStreamId(StreamId),
     Ping,
-    Blocked {
-        offset: u64,
-    },
-    StreamBlocked {
-        id: StreamId,
-        offset: u64,
-    },
-    StreamIdBlocked {
-        id: StreamId,
-    },
-    StopSending {
-        id: StreamId,
-        error_code: u16,
-    },
-    RetireConnectionId {
-        sequence: u64,
-    },
+    Blocked { offset: u64 },
```

Comment:
Unrelated style changes?

Reply:
rustfmt does this directly in response to the type extraction, for some reason.